src/routes/safe/components/Apps/components/ConfirmTxModal/index.tsx

```diff
@@ -50,6 +50,7 @@ export const ConfirmTxModal = (props: ConfirmTxModalProps): ReactElement => {
   const closeDecodedTxDetail = () => {
     hideDecodedTxData()
     props.onClose()
+    props.onTxReject(props.requestId)
```

Comment:
Maybe we should make a local function `onClose` and use it in both places where `props.onClose` is called?

Reply:
ok make sense, created a method and passed to `<ReviewConfirm />`